docs/3.0/tutorials/scraping.mdx

```diff
@@ -0,0 +1,503 @@
+
+    # ...
+
+
+@task
+def calculate_resolution_rate(issues: List[dict]) -> float:
+    """Task: Calculate the percentage of closed issues"""
+
+    # ...
+```
+
+<Expandable title="full example">
+Run the following code to see metrics calculation in action:
+
+```python
+from typing import List, Optional
+import httpx
+
+from prefect import flow, task
+
+
+@flow(log_prints=True)
+def analyze_repo_health(repos: List[str]):
+    """Analyze issue health metrics for GitHub repositories"""
+    for repo in repos:
+        print(f"Analyzing {repo}...")
+        
+        # Fetch and analyze all issues
+        issues = fetch_repo_issues(repo)
+        
+        # Calculate metrics
+        resolution_rate = calculate_resolution_rate(issues)
+        
+        print(f"Resolution rate: {resolution_rate:.1f}%")
+
+
+@flow
+def fetch_repo_issues(repo: str):
+    """Fetch all issues for a single repository"""
+    all_issues = []
+    page = 1
+    
+    for page in range(1, 3):  # Limit to 2 pages to avoid hitting rate limits
+        issues = fetch_page_of_issues(repo, page)
+        if not issues or len(issues) == 0:
+            break
+        all_issues.extend(issues)
+        page += 1
+
+    issue_details = []
+    for issue in all_issues[:5]:  # Limit to 5 issues to avoid hitting rate limits
+        issue_details.append(
+            fetch_issue_details.submit(repo, issue['number'])
+        )
+    
+    details = []
+    for issue in issue_details:
+        details.append(issue.result())
+
+    return details
+
+
+@task(log_prints=True)
+def fetch_page_of_issues(repo: str, page: int = 1) -> Optional[dict]:
+    """Fetch a page of issues for a GitHub repository"""
+    try:
+        response = httpx.get(
+            f"https://api.github.com/repos/{repo}/issues",
+            params={"page": page, "state": "all", "per_page": 100}
+        )
+        response.raise_for_status()
+        return response.json()
+    except Exception as e:
+        print(f"Error fetching issues for {repo}: {e}")
+        return None
+
+
+@task
+def fetch_issue_details(repo: str, issue_number: int) -> dict:
+    """Fetch detailed information about a specific issue"""
+    response = httpx.get(f"https://api.github.com/repos/{repo}/issues/{issue_number}")
+    issue_data = response.json()
+    
+    return issue_data
+
+
+@task
+def calculate_resolution_rate(issues: List[dict]) -> float:
+    """Calculate the percentage of closed issues"""
+    if not issues:
+        return 0
+    closed = sum(1 for issue in issues if issue['state'] == 'closed')
+    return (closed / len(issues)) * 100
+
+
+if __name__ == "__main__":
+    analyze_repo_health([
+        "PrefectHQ/prefect",
+        "tensorflow/tensorflow",
+        "huggingface/transformers"
+    ])
+```
+</Expandable>
+
+## Put it all together
+
+Here's the complete flow that combines all of these components.
+We'll also add retries, caching, and rate limiting to make the workflow more robust.
```

Comment:
Lots of goodies! I suggest linking each of the words to where the reader can learn more about these ideas, and adding a line about the benefit of each one might not be bad.